Source/FieldSolver/FiniteDifferenceSolver/EvolveB.cpp

```diff
@@ -0,0 +1,76 @@
+#include <WarpXAlgorithmSelection.H>
```

Comment:
WarpX-local includes with `""`, AMReX and other third party includes with `<>` :)

```suggestion
#include "WarpXAlgorithmSelection.H"
```